src/Symfony/Component/Serializer/Tests/Normalizer/Features/CallbacksTestTrait.php

```diff
@@ -165,6 +165,13 @@ public function provideNormalizeCallbacks()
                 ['bar' => '10-09-2011 06:30:00', 'foo' => null],
             ],
             'Collect a property' => [
+                [
+                    'bar' => static fn (array $bars) => implode('', $bars),
+                ],
+                [new CallbacksObject('baz'), new CallbacksObject('quux')],
+                ['bar' => 'bazquux', 'foo' => null],
+            ],
+            'Count a property' => [
```

Comment:
Something happened over here, `Count a property` should return `\count`, but it now returns previous `implode` version.

![image](https://github.com/symfony/symfony/assets/870747/94e2c65c-4127-4262-b5ac-fe01bf6af7ed)